example/hugow

```diff
@@ -63,15 +63,18 @@ fi
 DOWNLOAD_COMMAND=""
 DOWNLOAD_OUTPUT=""
 DOWNLOAD_SILENT=""
+DOWNLOAD_REDIRECT=""
 
 if command -v wget > /dev/null; then
     DOWNLOAD_COMMAND="wget"
     DOWNLOAD_OUTPUT="-O"
     DOWNLOAD_SILENT="-q"
+    DOWNLOAD_REDIRECT=""
```

Comment:
Flag to ensure redirects are followed for wget is `--mirror`

Reply:
Thanks @StephenBrown2 for you feedback, but the only thing I was able to find in the manual for `--mirror` was `Turn on options suitable for mirroring.` and also there's this:
```
--max-redirect=number
           Specifies the maximum number of redirections to follow for a resource.  The default is 20, which is usually far more than necessary. However, on those occasions where you want to allow more (or
           fewer), this is the option to use.
```
My point is, afaik _following redirect_ is enabled by default for `wget` and I'd really appreciate it if you can let me know what I might have missed.

Reply:
Ah, that should be correct, then.
I've been having troubles with wget receiving a 404, but curl gets the file fine, so I thought it might be related, but perhaps not.